sandbox/host/src/sandbox/wasmer_backend.rs

```diff
@@ -21,6 +21,7 @@
 use std::{cell::RefCell, collections::HashMap, rc::Rc};
 
 use wasmer::{Exportable, RuntimeError};
+use wasmer_types::TrapCode;
```

Comment:
doesn't wasmer reexport that?

Reply:
Cannot find this re-export. May be in future versions